lv_conf.h

```diff
@@ -1,6 +1,6 @@
 /**
  * @file lv_conf.h
- * Configuration file for v9.1.0
+ * Configuration file for v9.3.0-dev
```

Comment:
The new binding uses 9.3.0 release version, not the dev, but anyway thanks for updating this file.